### ISSUE TITLE
Improving the project

### ISSUE THREAD

**Issue by PythonMasterAlexander** [PythonMasterAlexander/portfolio-website#44]
Feedback from teacher after reviewing the project
- Please note that your images are lacking alt text
- The pages are also lacking meta descriptions, and I see that the Work.html page has a title that reads “About”
- While it is nice that I can see the github repo on your site, I would also like to see a thumbnail (or larger) of your netlify deployed sites that I can take a look at if I click on them

- Perhaps it is just my preference as well but I suggest that you take away the “social media” text before the icons, as they are self explanatory
